identity.html

```diff
@@ -0,0 +1,971 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+  <meta charset="utf-8">
+  <link href="webrtc.css" rel="stylesheet">
+  <title>Identity for WebRTC 1.0</title>
+  <script class="remove" src="respec-w3c-common.js" type="text/javascript">
+  // // keep this comment //
+  </script>
+  <script class="remove" src="identity.js" type="text/javascript">
+  // // keep
+    this comment //
```

Comment:
Problem?

Reply:
Copied verbatim from the existing draft. I was afraid there might be magic in the linebreak.